api/helpers/queryParamObjToStr.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * URLSearchParams does not support converting arrays to query params.
```

Comment:
```suggestion
 * URLSearchParams does not support converting arrays to separate query params.
```
It does support converting arrays to query params, just not how your app requires you to 🙂

Reply:
🤦